spec.md

```diff
@@ -2232,6 +2331,58 @@ message VolumeUsage {
   // Units by which values are measured. This field is REQUIRED.
   Unit unit = 4;
 }
+
+message VolumeHealth {
+  option (alpha_message) = true;
+
+  enum Condition {
```

Comment:
should we number these conditions with room to spare, in case we need to add intermediate values? at a glance these look like they are sorted by severity (minus UNKNOWN). if that's an idiom that we want to maintain for this enumeration, maybe number them like 10, 20, 30, 40 in case we think of a new value we want to squeeze in between?

or not .. this is really just food for thought, not a hard requirement IMO

Reply:
That's a good idea.  I'll use 10, 20, ...